### DISCUSSION TITLE
effect of required_multi_asset_neighbor() on normal assets

### DISCUSSION THREAD

**Discussion by yamhirotoCS** [dagster-io/dagster#19062] [Q&A] [answered]
Hey, we have some jobs whose assets use this method `required_multi_asset_neighbors()`as they have a Fivetran connector as a source. [**doc**](https://docs.dagster.io/_modules/dagster/_core/definitions/asset_selection#AssetSelection.required_multi_asset_neighbors)

Without it I get this following error when loading the job
```
_dagster._core.errors.DagsterInvalidSubsetError: When building job, the AssetsDefinition 'xxxx' contains asset keys [ ...list of assetKeys...]. This AssetsDefinition does not support subsetting. Please select all asset keys produced by this asset._
```

By looking in the code source, it looks like I could enable this feature on all the jobs by default. I tested on jobs that normally don't require this feature and nothing seems changed. Am I missing some edge cases that I should be aware of ?

**Comment by OwenKephart** [comment, marked as answer]
@yamhirotoCS There's no real downside to including this on all of your asset selections -- the main reason this is not enabled by default is just that it makes it less "silent" when you mistakenly attempt to materialize an impossible set of assets.

**Reply by yamhirotoCS** [reply]
Amazing @OwenKephart , thanks for confirming !
